internal/cli/require/require.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package require
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/tangzero/inflector"
+)
+
+// NoArgs returns an error if any args are included.
+func NoArgs(cmd *cobra.Command, args []string) error {
```

Comment:
these are the actual changes, the rest is just changing to use this package instead of teh cobra one

Reply:
😆 Now that's a ripple effect

Reply:
I mean, now we know we have a bout 150+ commands 🤷‍♂️